docs/content/languages/java.md

```diff
@@ -0,0 +1,226 @@
+---
+title: "Using Java with Fission JVM environment"
+weight: 10
+---
+
+Fission supports functions written in Java and runs then on JVM. Current JVM environment is based on openjdk8 and uses Spring boot as framework.
+
+## Before you start
+
+We'll assume you have Fission and Kubernetes setup.  If not, head over
+to the [install guide](../installation/_index.en.md).  Verify your Fission setup with:
+
+```
```

Comment:
Better to add type of code (here and other places) for syntax highlight.
```
```bash

```js
```

and `$` at the start of command will be great